OpenGeoscience/KTile
# Fixup test environment

This resolves some issues with running the test environment locally in both python 2.7 and python 3.4. It adds a docker-compose.yml file for bringing up postgis and memcached which are required for the tests to execute successfully.  

To test (from a fresh checkout of this branch):
```sh
cd tests
docker-compose up
```
in a new terminal (from the same checkout of this branch):
```sh
tox
```

For more detailed instructions see the [README](https://github.com/OpenGeoscience/ktile/blob/fixup-test-environment/tests/README.md)
